### PR TITLE
[Layout:Style] -- Fullscreen

### PR DESCRIPTION
**Pull request by maelswarm**

### CODE REVIEW

**Comment by obstropolos** [comment]
All the fullscreen expands look good to me! 
I would just ask that we lighten that dark dividing line for now. Otherwise once that's done, I'll merge in!

Screen Shot 2021-04-14 at 8 27 36 PM

**Review by mycodecrafting — APPROVED** [review]
This is much better than what we had! I would ideally like to see us using the browser's native fullscreen capabilities, but this is fine for now, and much better than what it currently is.

**Review by obstropolos — APPROVED** [review]

**Comment by obstropolos** [comment]
Will merge once conflicts are resolved

**Pull request merged**
